core/src/main/scala/pureconfig/package.scala

```diff
@@ -154,4 +261,14 @@ package object pureconfig {
       loadConfig[Config](resolvedTypesafeConfig)
     }
   }
+
+  /**
+   * @see [[loadConfigFromFiles]]
```

Comment:
@melrief Did you want to do anything with this comment? I'm just asking so it doesn't get lost.